server/src/test/java/org/apache/cassandra/sidecar/routes/OperationalJobHandlerTest.java

```diff
@@ -124,6 +124,8 @@ void testGetJobStatusRunningJob(VertxTestContext context)
               .expect(ResponsePredicate.SC_ACCEPTED)
               .send(context.succeeding(response -> {
                   assertThat(response.statusCode()).isEqualTo(ACCEPTED.code());
+                  LOGGER.info("Response Status:" + response.statusCode());
+                  LOGGER.info("Response Body:" + response.bodyAsString());
```

Comment:
nit: use `{}` to eval log message... I know that it is just test code. But for the sake of consistency.